app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+from app.errors import (NotVaccinatedError, NotWearingMaskError,
+                        OutdatedVaccineError)
```

Comment:
```suggestion
from app.errors import (
    NotVaccinatedError, 
    NotWearingMaskError,
    OutdatedVaccineError
)
```

Reply:
And here...
First I wrote imports to column, but "isort" changed my code )